flixel/FlxGame.hx

```diff
@@ -758,8 +759,6 @@ class FlxGame extends Sprite
 		}
 		FlxArrayUtil.clearArray(FlxG.swipes);
 		#end
-		
-		filters = filtersEnabled ? _filters : null;
```

Comment:
Seems like we should have a function called updateFilters or something then, because this is silly.